src/tyr/route_serializer_valhalla.cc

```diff
@@ -622,15 +622,28 @@ json::ArrayPtr legs(const valhalla::Api& api) {
 
 std::string serialize(const Api& api) {
   // build up the json object
-  auto trip_json = json::map({{"locations", locations(api.directions().routes(0).legs())},
-                              {"summary", summary(api)},
-                              {"legs", legs(api)},
-                              {"status_message", string("Found route between points")},
-                              {"status", static_cast<uint64_t>(0)}, // 0 success
-                              {"units", valhalla::Options_Units_Enum_Name(api.options().units())},
-                              {"language", api.options().language()}});
-  tyr::route_references(trip_json, api.trip().routes(0), api.options());
-  auto json = json::map({{"trip", trip_json}});
+  auto json = json::map({});
+  auto alternates = json::array({});
```

Comment:
this is where i added support for alternates in the valhalla route response format, pretty straight forward actually!